### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
- Added Login page
- Added Signup page
- Added Profile page
- Used RequiresAuth for the profile page

### Preview
[Link](https://deploy-preview-15--always-stream.netlify.app/)

### Note 
ignore the single-video commits its from another pr

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import Nav from "./Nav/Nav";
 import VideoCard from "./VideoCard/VideoCard";
 import Spinner from "./Spinner/Spinner";
 import Chips from "./Chips/Chips";
+import Input from "./Input/Input";
+import RequiresAuth from "./RequiresAuth/RequiresAuth";
```
Comment (dusariudaykumar) [inline src/components/index.js:11]: You can directly export the components from here.  You can avoid importing here and simply  use export

**Review by dusariudaykumar — APPROVED** [review]
The UI looks and the code looks good and clean. Good work !!

**Review by kumsomi — COMMENTED** [review]
- functionality is working fine
- Looks good to me
- Tho' you can refactor your code a little in order to make it clean

**Review by kumsomi — APPROVED** [review]

**Comment by SanketDhabarde** [comment]
> * functionality is working fine
> * Looks good to me
> * Tho' you can refactor your code a little in order to make it clean

Thanks for the suggestion. It will be really helpful if you suggest where I can clean up

File: src/components/index.js (hunk 1 above)
Comment (SanketDhabarde) [reply src/components/index.js:11]: I will try to look. if we can use it for the default export component. thanks for suggestion

**Pull request merged**